### PR TITLE
Ethereum blockchain unit tests

### PR DESCRIPTION
**Pull request by simonovic86**
- add more configuration options based on `NODE_ENV` variable
- implement anchor test on local (embedded) ganache instance
- add `husky` for `pre-commit` and `pre-push` git hooks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,9 +65,16 @@
       "@types/express": "^4.17.4",
       "@types/jest": "^25.2.1",
       "@types/node": "^13.11.0",
+      "@types/node-cron": "^2.0.3",
       "@types/uuid": "^7.0.2",
+      "@typescript-eslint/eslint-plugin": "^2.19.0",
+      "@typescript-eslint/parser": "^2.19.0",
       "babel-jest": "^25.1.0",
       "chai": "^4.2.0",
```
Comment (oed) [inline package.json:73]: I guess this can be removed?

Comment (oed) [reply package.json:73]: + chai as well?

File: package.json [hunk 2]
```diff
@@ -22,32 +22,38 @@
       "clean": "rm -rf ./build; rm -rf coverage; rm -rf .nyc_output",
       "start": "node ./build/app.js"
    },
+   "husky": {
+      "hooks": {
+         "pre-commit": "npm test",
+         "pre-push": "npm test"
```
[the remark this answers is not in the document]
Comment (simonovic86) [reply package.json:28]: makes sense 👍 I'll leave just `pre-commit`

File: package.json (hunk 1 above)
Comment (simonovic86) [reply package.json:73]: I'll remove those. As for `chai`, I've used its `assert` so I have to replace it with `Jest`s  `expect`.

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (simonovic86) [reply package.json]: actually, I left it for `pre-push` and added `npm run lint` for `pre-commit`. We'll have more commits and tests can last longer :)

File: package.json (hunk 1 above)
Comment (simonovic86) [reply package.json]: removed `chai` as well 👍

**Pull request merged**